### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by Daarken** [Skinz3/Symbioz#8]
Hi! I saw you are developing version 2.34. It will be free? If not, how much will it cost? Thanks ^^

**Comment by leonardo-spy** [comment]
skinz is rip, 2004-2016, may he rest in peace

**Comment by Skinz3** [comment]
2004 ? :3

Nope, 2.34 wont be free, i will fix a cost when the version will be stable ;)

**Comment by DoctorFrickle** [comment]
Why don't you host a server of your own? I can see this becoming very big, since you're obviously a competent developer and you'll know your code inside and out.

**Comment by Daarken** [comment]
Ok ^^ i hope it will be less than 100€ im studying Computer Engineering and i have to save money ^^' anyway thanks for your work. If i can help in something just ask for it!

**Comment by leonardo-spy** [comment]
Hello Skin , you do not have to make a cheaper price for me🤗? because I live in Brazil and here money is R$ (1 real =0,248943367 euros € and here the minimum wage ranges from R $ 880 to R $ 788) and to top it off I'm a student, not work, but if you do a cheap price to me I think I can pay 😁
thanks

**Comment by DoctorFrickle** [comment]
Dude stop being a suckup you're not special

**Comment by leonardo-spy** [comment]
Skin, you have prediction of when going out the 2:34 version ( to purchase ) ?

**Comment by DoctorFrickle** [comment]
June 23rd 2016
On May 29, 2016 3:21 PM, "leonardo-spy" <notifications@github.com> wrote:
>
> Skin, you have prediction of when going out the 2:34 version ( to
purchase ) ?

**Comment by leonardo-spy** [comment]
Yes, he said it would sell the version 2.34, but did not say a forecast date

**Comment by DoctorFrickle** [comment]
it was a prank dude just a social experiment lmao

**Comment by leonardo-spy** [comment]
I died

**Comment by simonswinslam** [comment]
I just finished my emulator dofus 2.30 everything is in it i think i will do like this team did and put it on github with sources.Btw I started from 0 and never used any other emulator.I will finish the spell effects left and put in some 2-4 days .

**Comment by Skinz3** [comment]
The 2.34 version will cost 250e :+1: 
i wont open server, because iits not actually my objective and i dont have money to finance it

**Comment by DoctorFrickle** [comment]
Servers finance themselves and actually make money though lol

**Comment by DanielAKUMA** [comment]
Hello Skins, what a beautiful work! Congratulations !
I just have a little issue with your Data Base : where are the WorldMapTrigger table to unlock the transitions portails ? 
Thanks :)

**Comment by leonardo-spy** [comment]
when you are going out the version 2.34?

**Comment by Skinz3** [comment]
"

Hello Skins, what a beautiful work! Congratulations !
I just have a little issue with your Data Base : where are the WorldMapTrigger table to unlock the transitions portails ?"
Thanks :)

This is not implanted in database, i will do it if you want, im hard working on the 2.34 version, this one wont be free :3, you can already buy it now ;)

**Comment by DanielAKUMA** [comment]
Je pense que tu es français :) je suis en train de vraiment apprendre le language #C en m'amusant :)
Tu développe un émulateur 2.34? Il risque de ne pas être stable non? Effectivement il manque plusieurs choses dans les bases de données de symbioz  :)
Les soleils de téléportation sont pas pris en charge et j'ai beaucoup de mal à comprendre les types des éléments id des interactives. Adobe crash souvent ^^

**Comment by DanielAKUMA** [comment]
Sinon comment on MP? ^^

**Comment by Skinz3** [comment]
Si, mais il sera privé ;)

**Comment by DanielAKUMA** [comment]
Ah mince donc on peux pas l'obtenir ^^' même en payant? Zut ^^.
Mais du coup on peux apprendre un peu en #C parce que j'aimerai insérer les mapttrigers, et des Additem sur les PNJ mais chaud ^^

**Comment by Daarken** [comment]
Skynz do you know how to debug roublard for stump 2.22? using a spell fixer i can summon bombes like if they are "monsters" but glyphs doenst work at all. If you have anything can you share it or sell it?

**Comment by leonardo-spy** [comment]
Skynz when going out version 2.34? (or 2.35...)

**Comment by leonardo-spy** [comment]
SKYNZ DIED?!!!

**Comment by Skinz3** [comment]
Its out, but its not free Leonardo

**Comment by leonardo-spy** [comment]
I already know it will not be for free, I want to buy

**Comment by leonardo-spy** [comment]
and 2:33 is by how many?

**Comment by Skinz3** [comment]
2.34 * 200e .

**Comment by leonardo-spy** [comment]
good

**Comment by leonardo-spy** [comment]
skinz vous êtes occupé maintenant?

**Comment by leonardo-spy** [comment]
because I wanted to know how is compressed to comprescell

**Comment by Skinz3** [comment]
Je suis en vacances ;) je reste dispo pour parler, voici mon skype: freak.dev

**Comment by leonardo-spy** [comment]
you do not answer me ,-,  call me there: leonardonarutobr (ignore the name ^^)

**Comment by leonardo-spy** [comment]
il y a une date pour le lancement?
